### ISSUE TITLE
Microscopy segmentation exps

### ISSUE THREAD

**Issue by NIRVANALAN** [NIRVANALAN/PyTorch_UNOdeMSegNet#1]
- Arch: UNet
- Training data: 512*512 patches of 14 slides(resized to 574 for UNet input)
- lr=0.01, momentum=0.99,epoch=50,batch=4
- loss: Pixel CE

## All data patches data split:
Trainset: 1.8k
Testset: 400 
## Remove pure BG images(without organelles)
Train: 1444
test: 362

## results on several dataset split:
settings | best ACC  | best loss
| --- | --- | ---
1000 unfiltered |  93.157772 | 0.292162 
800 unfiltered | 
1000 unfiltered, lr=0.02 | 93.151519 | 0.300175

**Comment by NIRVANALAN** [comment]
Considering the BG pixels (pixels that are not labeled) dominates the images, category CE loss may lead to overfitting(all pixels are predicted as BG pixels, and the ACC is still high), this time I tried with BCE+DICE Loss and trained models without BG pixels. The output of different class lies in different channel, and softmax as final layer activation function.

Also, 3 metrics are used to evaluate the model.
1. Pixel Accuracy
![image](https://user-images.githubusercontent.com/23359323/63218377-efed5980-c11e-11e9-92e7-9447a38b00c4.png)
2. Multi Class Pixel Accuracy
![image](https://user-images.githubusercontent.com/23359323/63218379-04315680-c11f-11e9-844b-2e86add81158.png)
3. **Multi Class IOU**
![image](https://user-images.githubusercontent.com/23359323/63218382-157a6300-c11f-11e9-98ad-556273594aa3.png)
